src/js/view/view.js

```diff
@@ -870,21 +870,15 @@ function View(_api, _model) {
             addClass(_playerElement, 'jw-flag-floating');
             _this.trigger(FLOAT, { floating: true });
             _model.set('floating', true);
+            _resizeOnFloat = true;
 
-            if (_isMobile) {
-                addClass(_wrapperElement, 'jw-float-top');
-            } else {
-                addClass(_wrapperElement, 'jw-float-bottom');
-                _resizeOnFloat = true;
-
-                // Resize within MAX_FLOATING_WIDTH×MAX_FLOATING_HEIGHT bounds, never enlarge.
-                const { width, height } = _this.getSafeRegion(false);
-                const floatingWidth = Math.min(width, MAX_FLOATING_WIDTH);
-                const floatingHeight = Math.min(height * floatingWidth / width, MAX_FLOATING_HEIGHT);
-                _this.resize(floatingWidth, floatingHeight, true);
+            // Resize within MAX_FLOATING_WIDTH×MAX_FLOATING_HEIGHT bounds, never enlarge.
+            const { width, height } = _this.getSafeRegion(false);
+            const floatingWidth = Math.min(width, MAX_FLOATING_WIDTH);
+            const floatingHeight = Math.min(height * floatingWidth / width, MAX_FLOATING_HEIGHT);
+            _this.resize(floatingWidth, floatingHeight, true);
```

Comment:
The third argument `resize(floatingWidth, floatingHeight, true)` is not being used. Did you mean to pass in `_resizeOnFloat`? That would be better (purer) than using this `_resizeOnFloat` flag.